resources/js/Pages/Character/Detail/CharacterDetail.tsx

```diff
@@ -1,26 +1,54 @@
-import React from 'react'
+import {Typography} from '@mui/material'
 import {ThemeProvider} from '@mui/material/styles'
+import {
+    CharacterCreateForm,
+    CharacterDetailBox,
+    CharacterDetailTable,
+    EditDrawer,
+} from 'Components'
 import {ApplicationLayout} from 'Layouts'
+import React, {useState} from 'react'
+import {CharacterRowData} from 'Types/character-row-data'
 import {getFontTheme} from 'Utils'
-import {CharacterDetailBox, CharacterDetailTable} from 'Components'
-import {CharacterDetailData} from 'Mock/character-detail-data'
 
-const CharacterDetail = () => {
-    const theme = getFontTheme('Form', 14)
+const theme = getFontTheme('Form', 14)
+
+const CharacterDetail = ({character, entries}: any) => {
+    const [isEditDrawerOpen, setIsEditDrawerOpen] = useState<boolean>(false)
+    const [editId, setEditId] = useState<number>()
```

Comment:
there is no need of setting edit id or data, since there is only one character in this page